pkgs/desktops/lxqt/qlipper/default.nix

```diff
@@ -12,7 +12,7 @@
 
 stdenv.mkDerivation {
   pname = "qlipper";
-  version = "5.1.2-unstable-2025-10-29";
+  version = "5.1.2";
```

Comment:
@romildo Isn't this misleading? We are still picking an unstable version here to work with Qt 6.10.